internal/trainings/trainings.go

```diff
@@ -1,13 +1,72 @@
 package trainings
 
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/ElenaMask/go1fl-sprint5-final/internal/personaldata"
+	"github.com/ElenaMask/go1fl-sprint5-final/internal/spentenergy"
+)
+
 type Training struct {
-	// TODO: добавить поля
+	Steps        int
+	TrainingType string
+	Duration     time.Duration
+	personaldata.Personal
 }
 
 func (t *Training) Parse(datastring string) (err error) {
-	// TODO: реализовать функцию
+	s := strings.Split(datastring, ",")
+	if len(s) != 3 {
+		return fmt.Errorf("invalid argument datastring = %s", datastring)
+	}
+	steps, err := strconv.Atoi(s[0])
+	if err != nil {
+		return err
+	}
+	if steps <= 0 {
+		return errors.New("количество шагов меньше или равно 0")
```

Comment:
озвучить так что все хорошо
```suggestion
		return errors.New("количество шагов не может быть меньше или равно 0")
```